dev-libs/nng/nng-1.10.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib multibuild
+
+DESCRIPTION="Light-weight brokerless messaging"
+HOMEPAGE="https://nng.nanomsg.org/"
+SRC_URI="https://github.com/nanomsg/nng/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+# compat and deprecated match upstream's default choice
+IUSE="+compat +deprecated doc ssl static-libs test tools"
```

Comment:
Let's please [not](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302) add static-libs unless a reverse dependency needs it.

Reply:
Done.